src/rules/enforce-firestore-facade.ts

```
@@ -92,6 +106,51 @@ const isFirestoreAssignment = (node: TSESTree.Node): boolean => {
   return false;
 };
 
+const handleAssignmentExpression = (node: TSESTree.AssignmentExpression): void => {
```

ESLint

[error] Replace `node:·TSESTree.AssignmentExpression` with `⏎··node:·TSESTree.AssignmentExpression,⏎`

(prettier/prettier)